test/class.js

```diff
@@ -11,6 +11,38 @@ describe('Class', function() {
     });
   });
 
+  describe('#cid', function() {
+    beforeEach(function() {
+      this.instance = new Metal.Class();
+    });
+
+    it('should contains cid', function() {
+      expect(this.instance).to.include.keys('cid');
```

Comment:
Could you assert what `cid` is? Something like:

```js
expect(this.instance).to.have.property('cid').that.match(/^metal[0-9]+$/)
```

Also another test that asserts that `cid`'s are different.